modules/test/playwright/tests/frontend-css/main/cadmin.spec.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../../fixtures/featureFlagsTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import getRandomString from '../../../utils/getRandomString';
+import getPageDefinition from '../../layout-content-page-editor-web/main/utils/getPageDefinition';
+import {frontendCssTest} from './extensions/frontendCssTest';
+
+const test = mergeTests(
+	apiHelpersTest,
+	frontendCssTest,
+	featureFlagsTest({
+		'LPS-178052': {enabled: true},
+	}),
+	loginTest()
+);
+
+const DEFAULT_COLOR = 'rgb(48, 49, 63)';
+const OVERRIDE_COLOR = 'rgb(0, 255, 0)';
+const ELEMENT_SELECTOR = '.cadmin .control-menu-level-1-dark';
+
+const SMALLER_PRECEDENCE_CSS = `${ELEMENT_SELECTOR} { background-color: ${OVERRIDE_COLOR}; }`;
+const HIGHER_PRECEDENCE_CSS = `${ELEMENT_SELECTOR} { background-color: ${OVERRIDE_COLOR} !important; }`;
```

Comment:
Keep these as local variables since they are only used once. So that we can see the CSS to understand what higher and lower precendence means.